checklists/coding-life-cycle-of-a-jira-issue.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Coding Life-Cycle of a Jira issue
+category: General
+date: "2020-03-12"
+tags: ['jira', 'issue', 'development']
+description: You have some protocols before you start to coding, on process and afterwards
+---
+
+- [ ] Read acceptance criteria carefully  
+Be sure all cases covered. Even you already discussed the issue with the team, some new cases may come up your mind. Don't hesitate to discuss AC one more time.
+
+- [ ] Create a branch for the issue  
+You should create a branch with name `feature/ABC-123_brief-description`. For bugs change `feature` with `bug` keyword.
+
+- [ ] Push your branch to remote immediately
+Since we have Jira-GitHub integration, you have to push the branch to remote in order to trigger "In Progress" transition
```

Comment:
```suggestion
Since we have Jira-GitHub integration, you have to push the branch to remote in order to trigger "In Progress" transition on Jira.
```